src/common/plus/subscription.ts

```diff
@@ -0,0 +1,16 @@
+export enum SubscriptionProvider {
+  Paddle = 'paddle',
+  AppleStoreKit = 'storekit',
+}
+
+export enum SubscriptionProcessor {
+  Cores = 'cores',
```

Comment:
Cores is not really subscription it, is more ProductType for this enum?

Reply:
You're right, will rename it

Reply:
I renamed the `SubscriptionProcessor` enum to `PurchaseType`.